### ISSUE TITLE
Images and internal links don't work?

### ISSUE THREAD

**Issue by chris-bingham** [marcbachmann/node-html-pdf#508]
I am unable to get images to work in the generated PDF, if i prefix the base value with `file://` i get a broken image, otherwise I just get the image alt text.

I have links within the HTML to various elements within the same page, these don't work and are prefixed with the value of the `base` property. 

Any help with images would be greatly appreciated, and is it possible to have internal links?

Heres my code:

```javascript
var html = fs.readFileSync(htmlName, "utf8");
var options = {
    phantomPath: require("phantomjs-prebuilt").path,
    base: "file://" + path.resolve(".")
};

pdfFromHtml.create(html, options).toFile(pdfName, function(err, res) {});
```

**Comment by Serhioromano** [comment]
Try to save your HML to file first and open that file from the folder you saved it and see if you see images. It is obviously a problem with images path.

**Comment by Joshmamroud** [comment]
Try ending the base path with a "/". That worked for me.

**Comment by btlakal** [comment]
Please let me know if anyone got the solution for the internal links
